mRides-app/Mappers/UserMapper.cs

```diff
@@ -146,5 +146,21 @@ public void setGender(int id, string newGender)
         {
             userGateway.setGender(id, newGender);
         }
+
+        /**
+        * Set the theme of an activity depending on whether the user is a driver or a rider
+        */
```

Comment:
Don't forget to change the default comments style to XML so it can be used in the generated documentation